addons/binding/org.openhab.binding.knx/ESH-INF/thing/ip.xml

```diff
@@ -0,0 +1,268 @@
+            <channel id="errors5min" typeId="errors" />
+        </channels>
+        
+        <config-description>
+            <parameter name="ipAddress" type="text">
+                <label>Network Address</label>
+                <description>Network address of the KNX/IP gateway</description>
+                <required>true</required>
+            </parameter>
+            <parameter name="portNumber" type="integer">
+                <label>Port Number</label>
+                <description>Port number of the KNX/IP gateway</description>
+                <required>true</required>
+            </parameter>
+            <parameter name="localIp" type="text">
+                <label>Local Network Address</label>
+                <description>Network address of the local host to be used to set up the connection to the KNX/IP gateway</description>
+                <required>false</required>
+            </parameter>      
+            <parameter name="ipConnectionType" type="text">
+                <label>IP Connection Type</label>
+                <description>The ip connection type for connecting to the KNX bus. Could be either TUNNEL or ROUTER</description>
+                <required>true</required>
+            </parameter>  
+            <parameter name="localSourceAddr" type="text">
+                <label>local KNX GA</label>
+                <description>The group address for identification of this KNX/IP gateway within the KNX bus. Default is 0.0.0</description>
+                <required>false</required>
+            </parameter>
+            <parameter name="readingPause" type="integer">
+                <label>Reading Pause</label>
+                <description>Time in milliseconds of how long should be paused between two read requests to the bus during initialization</description>
+                <required>false</required>
+                <default>50</default>
+            </parameter>
+            <parameter name="responseTimeOut" type="integer">
+            	<label>Response Time Out</label>
+            	<description>Timeout in milliseconds to wait for a response from the KNX bus</description>
+            	<required>false</required>
+            	<default>10000</default>
+        	</parameter>  
+        	<parameter name="readRetriesLimit" type="integer">
+            	<label>Read Retries Limit</label>
+            	<description>Limits the read retries while initialization from the KNX bus</description>
+            	<required>false</required>
+            	<default>3</default>
+        	</parameter>
+        	<parameter name="autoReconnectPeriod" type="integer">
+            	<label>Auto Reconnect Period</label>
+            	<description>Seconds between connect retries when KNX link has been lost, 0 means never retry</description>
+            	<required>false</required>
+            	<default>0</default>
+        	</parameter> 
+        </config-description>
+    </bridge-type>
+    
+    <channel-type id="errors">
```

Comment:
I'd suggest to move the definition of all channel type to a separate file (e.g. channels.xml), since you anyhow seem to reuse them across the different things (currently you declare them multiple times, which is not valid).